README.md

```diff
@@ -109,8 +109,12 @@ type PropFilter = (prop: PropItem, component: Component) => boolean;
 
 const options = {
   propFilter: (prop: PropItem, component: Component) => {
-    if (prop.parent) {
-      return !prop.parent.fileName.includes("node_modules");
+    if (prop.parents !== undefined && prop.parents.length > 0) {
```

Comment:
I guess the `props.parents` should be `props.declarations` now.

Reply:
Thank you, fixed it 👍